dev/TeachingTip/TeachingTip.cpp

```diff
@@ -199,7 +197,28 @@ void TeachingTip::OnPropertyChanged(const winrt::DependencyPropertyChangedEventA
             TeachingTipTestHooks::NotifySubtitleVisibilityChanged(*this);
         }
     }
+    else if (property == s_ActionButtonContentProperty)
+    {
+        UpdateButtonsState();
+        winrt::IInspectable value = unbox_value<winrt::IInspectable>(args.NewValue());
```

Comment:
>unbox_value [](http://example.com/codeflow?start=36&length=11)

Why the need to call unbox_value here? I think arg.NewValue() is already IInspectable?